sim/core/statweight.go

```diff
@@ -301,6 +313,7 @@ func CalcStatWeight(swr proto.StatWeightsRequest, statsToWeigh []stats.Stat, ref
 	return result
 }
 
+// TODO: Get rid of this?
```

Comment:
This was once used in an attempt to properly calculate standard deviations. The current code uses much simpler logic but I don't think either method is fully correct.

Could probably delete this if you want, we can always recover it from the TBC code if we want it back.

Reply:
I'll clean it up, and I agree, I don't think the current way to determine deviations is correct, but it works for now. I don't particularly find EPs/weights attractive anyway